### PR TITLE
Add query-results dir to gitignore

### PR DESCRIPTION
**Pull request by jonnyhork**
### What does this PR do?
This PR will add a new `query-results` directory to the `.gitignore` of the project template.
As part of the new soql query builder extension, customers will be able to save results from a soql query to the disk however we don't anticipate they will want to check in those results to source control by default.
### What issues does this PR fix or reference?
@W-8136966@

### CODE REVIEW

File: packages/templates/src/templates/project/gitignore [hunk 1]
```diff
@@ -12,6 +12,9 @@
 # LWC Jest coverage reports
 coverage/
 
+# SOQL Query Results
+**/scripts/soql/query-results
```
Comment (jonnyhork) [inline packages/templates/src/templates/project/gitignore:16]: I chose to be more explicit with the target directory incase a customer wants to create their own `query-results` dir that is checked in, we don't exclude that name from being used everywhere.

**Comment by jonnyhork** [comment]
From what I can tell the `scripts` directory is not included in when pushing source to an org, so I did not change the .forceignore but please correct me if I am missing something.

**Review by mysticflute — APPROVED** [review]

File: packages/templates/src/templates/project/gitignore (hunk 1 above)
Comment (brpowell) [reply packages/templates/src/templates/project/gitignore:16]: Are results automatically saved to this path?

**Comment by brpowell** [comment]
Can you change the base branch to target `develop` instead of `main`?

@lcampos we might need to update the default branch on this repo to develop.

**Comment by lcampos** [comment]
> Can you change the base branch to target `develop` instead of `main`?
> 
> @lcampos we might need to update the default branch on this repo to develop.

@brpowell done

File: packages/templates/src/templates/project/gitignore (hunk 1 above)
Comment (jonnyhork) [reply packages/templates/src/templates/project/gitignore:16]: yes results are automatically saved to this path.

**Review by jonnyhork — COMMENTED** [review]
@brpowell 
Sorry about the mix up with main, I changed the target branch to develop and pulled that into my feature branch but there still seems to be some weirdness going on.

It might be easier for me to just make a new PR unless the changes you see here are ok?

**Comment by brpowell** [comment]
@jonnyhork yeah because the commits on main are technically different you'll have to make a new branch based off of develop, sorry for the mix up. You should be able to cherry-pick the commit from this branch and open a new PR.

**Comment by brpowell** [comment]
@lcampos we might also need to add Jonny as a contributor so the builds run

**Comment by jonnyhork** [comment]
New PR at https://github.com/forcedotcom/salesforcedx-templates/pull/227

**Pull request closed without merging**